os/board/rtl8721csm/src/component/common/file_system/ftl/ftl.c

```diff
@@ -1406,6 +1514,26 @@ uint32_t ftl_save(void *pdata, uint16_t offset, uint16_t size)
 }
 #endif
 
+#if defined(CONFIG_AMEBAD_TRUSTZONE) && (FTL_SECURE == 1)
+/* FTL Secure Init */
+uint32_t ftl_secure_init(void)
+{
+	uint32_t ret = 0;
+	ns_ftl_func_struc in_func;
+
+	/* Prepare non-secure API to use in Secure */
+	in_func.device_lock = device_mutex_lock;
+	in_func.device_unlock = device_mutex_unlock;
+	in_func.setstatusbits = ftl_ns_setstatusbits;
+	in_func.save_to_storage = ftl_save_to_storage_i;
+	in_func.load_from_storage = ftl_load_from_storage_i;
+
+	up_allocate_secure_context(FTL_SECURE_CONTEXT_SIZE);
+	ret = ameba_ftl_secure_init(FTL_KEY_ADDR, &in_func);
+	up_free_secure_context();
+}

```

Comment:
The return type of this function is uint32_t. I think you need to add `return ret`.

Reply:
Added return.